src/melee/it/inlines.h

```diff
@@ -10,9 +14,33 @@ static inline Item* GetItemData(HSD_GObj* item_gobj)
     return item_data;
 }
 
-static inline Item* GetItemDirect(HSD_GObj* item_gobj)
+static inline void HSD_JObjSetFacingDirItem(HSD_JObj* jobj, Item* it)
 {
-    return item_gobj->user_data;
+    if (it->xDC8_word.flags.x19 == true)
+        HSD_JObjSetRotationY(jobj, M_PI / 2 * it->facing_dir);
+}
+
+static inline void ItUnkHoldKind(HSD_GObj* item_gobj)
+{
+    Item* it = (Item*) HSD_GObjGetUserData(item_gobj);
+
+    switch (it->hold_kind) {
+    case 4:
+    case 5:
+    case 6:
+    case 7: {
+        int temp_r3 = func_8017E068();
+        if (temp_r3 >= 0)
+            it->xC3C = lbl_804D6D28->x80_float[temp_r3];
+    }
+    }
+}
+
+static inline void HSD_JObjSetScaleItem(Item* it, HSD_JObj* jobj, Vec3* scl)
```

Comment:
these can all stay in item.c

Reply:
I was hoping you wouldn't notice so I could move them later when I'm not being lazy but ok